eng/vcpkg.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "azure-sdk-for-rust",
+  "dependencies": [
+    "openssl"
+  ]
+}
```

Comment:
Missing crlf at end of file.

Reply:
That's odd. I wonder what editor I used that would've ignored `.editorconfig` or just didn't do it outright. VSCode (with EditorConfig extension), helix (my personal settings always cause a newline), and VS (supports `.editorconfig` out of the box) all do.

Reply:
That said, I don't think it necessary to get re-approvals for adding that. If someone edits in the future, it should happen then. I'm just puzzled how it didn't in the first place.